.travis.yml

```diff
@@ -1,6 +1,5 @@
 language: c
 sudo: required
-install: wget https://raw.githubusercontent.com/ocaml/ocaml-ci-scripts/master/.travis-opam.sh
```

Comment:
Removing this line breaks the [instructions](/ocaml/ocaml-ci-scripts/blob/master/README-travis.md#opam-package-travis-opamsh):
> 2. Copy [.travis.yml](https://raw.githubusercontent.com/ocaml/ocaml-ci-scripts/master/.travis.yml) at the root of your project.

Reply:
Hmm - perhaps `test -e .travis.opam || wget ...`